LiskHQ/lisk-desktop
# Indicate unclaimed rewards

### What was the problem?

This PR resolves #5064

### How was it solved?

- [x] Implement required UI
- [x] integrated dry run on stake form
- [x] Extend unit tests

### How was it tested?

- Add, Edit or remove a stake from/to a validator
- Navigate to the staking queue modal
- Click on the `Contine` button
- Staked validators with rewards should be visible
